src/transformers/models/whisper/feature_extraction_whisper.py

```diff
@@ -131,6 +131,27 @@ def _torch_extract_fbank_features(self, waveform: np.array) -> np.ndarray:
         log_spec = (log_spec + 4.0) / 4.0
         return log_spec.numpy()
 
+    def _torch_extract_fbank_features_batch(self, waveforms: np.array, device: str) -> np.ndarray:
```

Comment:
I don't think we need a new function we can update the original one no?

Reply:
@ArthurZucker,
I've committed the recent changes for your review. Please take a look.

Additionally, I've observed a slight discrepancy in the results when computing features on the GPU. The mismatch is on the order of 1e-5 and is primarily attributed to differences in the Short-Time Fourier Transform (STFT) computation results. Can you suggest what should I do here?

Reply:
it's alright IMO, as long as it translates to 1e-5 in the output logits!